src/script/backport-create-issue

```diff
@@ -0,0 +1,230 @@
+                relation['relation_type'] + "' to " + self.url(other))
+            success = False
+            continue
+        release = get_release(other)
+        existing_backports.add(release)
+        logging.debug(url(issue) + " backport to " + release + " is " +
+            redmine_endpoint + "/issues/" + str(relation['issue_to_id']))
+    if existing_backports == issue['backports']:
+        logging.debug(url(issue) + " has all the required backport issues")
+        return success
+    if existing_backports.issuperset(issue['backports']):
+        logging.error(url(issue) + " has more backport issues (" +
+            ",".join(sorted(existing_backports)) + ") than expected (" +
+            ",".join(sorted(issue['backports'])) + ")")
+        return False
+    backport_tracker_id = tracker2tracker_id['Backport']
+    for release in issue['backports'] - existing_backports:
+        if release not in releases():
+            logging.error(url(issue) + " requires backport to " +
+                "unknown release " + release)
+            success = False
+            break
+        subject = release + ": " + issue['subject']
+        if dry_run:
+            logging.info(url(issue) + " add backport to " + release)
+            continue
+        other = r.issue.create(project_id=issue['project']['id'],
+                               tracker_id=backport_tracker_id,
+                               subject=subject,
+                               priority='Normal',
+                               target_version=None,
+                               custom_fields=[{
+                                   "id": release_id,
+                                   "value": release,
+                               }])
+        r.issue_relation.create(issue_id=issue['id'],
+                                issue_to_id=other['id'],
+                                relation_type='copied_to')
+        logging.info(url(issue) + " added backport to " +
+                     release + " " + url(other))
+    return success
+
+def iterate_over_backports(r, issues, dry_run):
+    counter = 0
+    for issue in issues:
+        counter += 1
+        logging.debug("{} ({}) {}".format(issue.id, issue.project,
+            issue.subject))
+        print('{}\r'.format(issue.id), end='', flush=True)
+        if not has_tracker(r, issue['project']['id'], 'Backport'):
+            logging.info("{} skipped because the project {} does not "
+                "have a Backport tracker".format(url(issue),
+                issue['project']['name']))
+            continue
+        if not set_backport(issue):
+            logging.error(url(issue) + " no backport field")
+            continue
+        if len(issue['backports']) == 0:
+            logging.error(url(issue) + " the backport field is empty")
+        update_relations(r, issue, dry_run)
+    logging.info("Processed {} issues with status Pending Backport"
+                 .format(counter))
+    return None
```

Comment:
this return value is never used. and a method return nothing actually evaluates to `None`. so this is not necessary.

Reply:
I prefer to make the return value explicit. When I'm re-reading the code, it signals to me that this function is not designed to return anything useful.

Reply:
While I support not returning, I'm ok just merging this as is as this should beat copying issues in tracker that many users get wrong anyway

Reply:
@theanalyst Thanks! The reasoning for making the return value explicit is (a) it does something useful (remind me that the function is only used for its side effects, not for its return value), and (b) it does no harm. But this is a matter of coding style preference.